syntax = "proto3";

import "google/protobuf/empty.proto";

service DisplayService {
    rpc DisplayText (DisplayTextRequest) returns (google.protobuf.Empty) {}
}

message DisplayTextRequest {
    string text = 1;
}